config/secrets.yml

```diff
@@ -5,8 +5,10 @@ development:
 test:
   secret_key_base: secret
   jwt_auth_secret: secret
+  notify_non_production_allow_list: ["allowed@example.com"]
 
 production:
   secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>
   jwt_auth_secret: <%= ENV['JWT_AUTH_SECRET'] %>
   notify_api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] %>
+  notify_non_production_allow_list: <%= ENV['GOVUK_NOTIFY_ALLOW_LIST'] %>
```

Comment:
I don't think this is a secret so probably belongs in config/application.rb

Reply:
Also note that that this would be a string rather than an array unlike the one in test. I think it'd be better to treat it as a string though as I don't think we're expecting a need for multiple recipients.

Reply:
I've removed this and now referencing the env var directly in MailRecipientInterceptor.  I'll make sure the email is added as a string in Puppet 👍